### ISSUE TITLE
PBMT : SMT...?

### ISSUE THREAD

**Issue by bittlingmayer** [machinetranslate/machinetranslate.org#157]
These are often used synonymously, because PBMT was the last major wave of SMT.

Does it deserve an article?  Or a section in SMT?  Or to be treated as a synonym?

**Comment by bittlingmayer** [comment]
@zouharvi @randyscansani

**Comment by zouharvi** [comment]
I think it's still interesting (especially because it gave us word-alignment etc) but I don't know if it still has relevance in the industry? Maybe something with phrase-level translation memories?

**Comment by cefoo** [comment]
The Google Translate API for Trados 2019 allows you to choose the system you want to use: PBMT or NMT.
![imagen](https://user-images.githubusercontent.com/33723604/149557457-ad8df0f6-f7e6-4d10-bb0a-80ef20fa402c.png)

As @zouharvi mentions, this is something that can be found in the translation industry still to this day.
For our translators' audience, I think it may be worth to further describe PBMT. Maybe not in an article of its own (perhaps redirecting can be possible?), but inside the SMT article.

**Comment by randyscansani** [comment]
I agree that PBMT can still be relevant. It's probably not used extensively in the industry, but so are some other topics in the roadmap (like unsupervised MT). In general, anyone who wants to know about MT and has already read articles/manuals on the topic has probably stumbled upon the PBMT acronym, so it might be worth adding a few words about it in my opinion.

**Comment by bittlingmayer** [comment]
Google Cloud just uses "PBMT" to mean the last generation of SMT, e.g. https://stackoverflow.com/a/63787121/4486860.

**Comment by zouharvi** [comment]
Seeing this discussion, maybe this is a call for us to have a disambiguating paragraph on RBMT, SMT, PBMT & NMT.

**Comment by zouharvi** [comment]
@bittlingmayer @cefoo @randyscansani Was this issue resolved?

**Comment by cefoo** [comment]
@zouharvi We have a very short list describing the different approaches to SMT in the [SMT article](https://machinetranslate.org/statistical-machine-translation). Perhaps it's a good basis for expanding on the topic.

**Comment by bittlingmayer** [comment]
My concern is that people searching for PBMT will not find our resources, even though we cover it.

I think we can help a bit by modifying the Approaches text, from

> By the 2010s, the top systems, like Google Translate, used statistical machine translation. By the 2020s, the top systems used neural machine translation.

to

> By the 2010s, the top systems, like Google Translate, used statistical machine translation (SMT).
> Statistical machine translation is also known as phrase-based machine translation (PBMT).
>
> By the 2020s, the top systems used neural machine translation (NMT).

**Comment by bittlingmayer** [comment]
If we really want to be correct about NMT being a type of SMT, we can do that.
